spring-grpc-core/src/main/java/org/springframework/grpc/server/DefaultGrpcServerFactory.java

```diff
@@ -23,16 +23,11 @@
 
 import com.google.common.collect.Lists;
 
-import io.grpc.Grpc;
-import io.grpc.InsecureServerCredentials;
-import io.grpc.Server;
-import io.grpc.ServerBuilder;
-import io.grpc.ServerCredentials;
-import io.grpc.ServerProvider;
-import io.grpc.ServerServiceDefinition;
+import io.grpc.*;
```

Comment:
Please remove the wildcard imports. We need to update our formatter to balk about this.